images/windows/editor/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG baseImage="unityci/base:windows-latest"
+ARG hubImage="unityci/hub:windows-latest"
+
+#############
+#  Builder  #
+#############
+
+FROM $hubImage as Builder
+
+# Install Editor
+# The portion after the & is to change the exit code to 0 if we exited 1 from 
+# the installation. Otherwise docker believes there was an error
+ARG version
+ARG changeSet
+RUN "C:/Program Files/Unity Hub/Unity Hub.exe" -- --headless install \
+                                               --version %version% \
+                                               --changeset %changeSet% \
+                                               --module windows-il2cpp \
+                                               & if %ERRORLEVEL% EQU 1 exit 0
+
+# TODO: Handle additional modules
+
```

Comment:
module should be passed in through the workflow (matrix in new_build flow or retry args in retry flow)